client/test/e2e/patient/invoice.spec.js

```diff
@@ -30,7 +30,7 @@ describe('patient invoice', function () {
     browser.get(path);
   });
 
-  it('invoices a patient for a single item', function () {
+  it.skip('invoices a patient for a single item', function () {
```

Comment:
Are these intentionally left skipped? I assume with these should be re-implemented by the person that does the patient invoice posting core.